test/reduceRight.js

```diff
@@ -21,7 +21,7 @@ describe('reduceRight', function() {
     var something = R.reduceRight(avg, 54);
     var rcat = R.reduceRight(R.concat, '');
     eq(something([12, 4, 10, 6]), 12);
-    eq(rcat(['1', '2', '3', '4']), '4321');
+    eq(rcat(['1', '2', '3', '4']), '1234');
   });
```

Comment:
Similarly here.  I think the existing test cases no longer show anything of particular use.  But one that demonstrates the tree structure would be useful.